scipy/optimize/_minpack_py.py

```diff
@@ -639,6 +641,19 @@ def curve_fit(f, xdata, ydata, p0=None, sigma=None, absolute_sigma=False,
         `mesg`, and `ier`.
 
         .. versionadded:: 1.9
+    nan_policy : {'propagate', 'raise', 'omit', None}, optional
+        Defines how to handle when input contains nan.
+        The following options are available (default is 'None'):
+
+          * 'propagate': just propagate nan values
```

Comment:
This is not very specific about how `'propagate'` is _supposed_ to work in `stats`. In `stats`,

> `'propagate'`: if a NaN is present in the axis slice (e.g. row) along which the statistic is computed, the corresponding entry of the output will be NaN.

That is not always what you get by simply ignoring that NaNs might be present.

What does this mean for `curve_fit`?